dev-tools/ecs-migration.yml

```diff
@@ -0,0 +1,29 @@
+# The ECS migration file contains the information about all the Beats fields which are migrated to ECS in 7.0.
+# The goal of the file is to potentially have scripts on top of this information to convert visualisations and templates
+# based on this information in an automated way and to keep track of all changes which were applied.
+#
+# The format of the file is as following:
+#
+# - from: source-field-in-6.x
+#   to: target-filed-in-ECS
+#   # Alias field is useful for fields where there is a 1-1 mapping from old to new
+#   alias: true-if-alias-is-required-in-6x
+#   # Copy to is useful for fields where multiple fields map to the same ECS field
+#   copy_to: true-if-field-should-be-copied-to-target-in-6x
+#
+
+
+- from: beat.name
```

Comment:
@bleskes This is the file I have in mind to track the migration of the fields so it can be used later to do automatic migrations.

Reply:
👍

Reply:
What's the meaning of this structure? I'm wondering about the direction of the alias this is suggesting:

1. Does it mean field `offset` is type alias, and points to the data in `log.offset`?
  * This would be appropriate if we want to start preparing for ECS in 6.x 
2. Does it mean field `log.offset` is type alias, and points to the data in `offset`?
  * This will be appropriate for 7.x, for backwards compatibility

Since we may actually do both, I think we should rename the keys of the structures slightly.

I currently read them as "old field is in `from`" and "new field is in `to`". However when you read with the strategy in mind, it reads like "alias from offset to log.offset", which is confusing.

What do you think about the following?

```YAML
- old: offset
  new: log.offset
  strategy: alias # or copy_to
```

Reply:
I messed up the meaning of which strategy made sense in which version. Updated my comment above :)

Reply:
Not a blocker for this PR, just to be clear. Just maybe a thing to rethink in a separate PR

Reply:
The structure can still change, what is important right now is that we have a structure way of documenting it. We can focus on naming later.